src/main/java/com/nike/cerberus/operation/composite/CreateCmsClusterOperation.java

```diff
@@ -0,0 +1,46 @@
+package com.nike.cerberus.operation.composite;
+
+import com.beust.jcommander.internal.Lists;
+import com.nike.cerberus.command.cms.CreateCmsAsgCommand;
+import com.nike.cerberus.command.composite.CreateCmsClusterCommand;
+import com.nike.cerberus.command.core.CreateInstanceProfileCommand;
+
+import java.util.List;
+
+public class CreateCmsClusterOperation extends CompositeOperation<CreateCmsClusterCommand> {
+
+    @Override
+    protected List<ChainableCommand> getCompositeCommandChain(CreateCmsClusterCommand compositeCommand) {
+        return Lists.newArrayList(
+                new ChainableCommand(new CreateInstanceProfileCommand()),
```

Comment:
This also needs the slack delegate treatment, I think.

Reply:
I think an argument could be made that to keep behavior of the `create-cms-cluster` command the same as it used to be, when the instance profile was created along with the cluster, there shouldn't be any instance profile parameter provided. The new instance profile command does take in an optional parameter, but more so for individual use of the command.